scripts/ironic/playbooks/ironic-pre-config.yml

```diff
@@ -0,0 +1,112 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+- name: Pre install config for ironic
+  hosts: localhost
+  tasks:
+    - name: Create ironic network configurations
+      blockinfile:
+        dest: /etc/network/interfaces.d/osa_ironic_interfaces.cfg
+        create: yes
+        block: |
+          auto eth2
+          auto br-ironic-ipmi
+          iface br-ironic-ipmi inet static
+              bridge_ports eth2
+              bridge_stp off
+              bridge_waitport 10
+              bridge_fd 0
+              offload-sg off
+              address 172.29.228.100
+              netmask 255.255.252.0
+
+          auto eth3
+          iface eth3 inet manual
+          auto br-tftp
+          iface br-tftp inet static
+              bridge_stp off
+              bridge_waitport 10
+              bridge_fd 0
+              offload-sg off
+              bridge_ports eth3
```

Comment:
If we change *eth3* to *none* we could make this same patch work as-is without needing the additional cloud networks.